containers/hugo/README.md

```diff
@@ -0,0 +1,39 @@
+# Hugo
+
+## Summary
+
+*Develop static sites with Hugo, includes everything you need to get up and running.*
+
+|          Metadata           |                    Value                     |
+| --------------------------- | -------------------------------------------- |
+| *Contributors*              | [Aaryn Smith](https://gitlab.com/aarynsmith) |
+| *Definition type*           | Dockerfile                                   |
+| *Works in Codespaces*       | Yes                                          |
```

Comment:
I don't think VS Codespaces (or Codespaces in GitHub) works with Alpine based containers at the moment. Were you able to verify this works?

Reply:
You are correct, I thought I had worked when I tested it, but I guess it didn't. Updating now.